mlflow/catboost.py

```diff
@@ -272,7 +272,7 @@ def _load_pyfunc(path):
     )
 
 
-def load_model(model_uri):
+def load_model(model_uri, artifact_path=None):
```

Comment:
I think `dst_path` or `path` might be better here to distinguish from the `artifact_path` parameter of `log_model()`. This will make it clearer that `artifact_path` is not related to the model_uri / run.

```suggestion
def load_model(model_uri, dst_path=None):
```

Reply:
+1 for this change. `download_artifacts` also has a `dst_path` argument:

https://github.com/mlflow/mlflow/blob/8bd75032de3a2ee6bde42a030b72b5a68ab990ce/mlflow/tracking/client.py#L1353-L1365

Reply:
+1